python/cudf/cudf/core/series.py

```diff
@@ -1635,6 +1636,71 @@ def applymap(self, udf, out_dtype=None):
           * math.gamma()
           * math.lgamma()
 
+        * StringColumns are not supported in `applymap` method.
```

Comment:
:fire: Fantastic docs :)

`StringColumn` should be considered a detail that users shouldn't know or care about. Better we say something along the lines of "Columns containing strings are not supported" or "string dtypes are not supported".

Reply:
Sounds right, Done 👍